src/vde_backend/compute_power.py

```diff
@@ -296,11 +296,42 @@ def recompute_power_data(last_config, dates=None):
 # ====================================================================================
 
 
+def process_arguments():
+    """
+    Function to process arguments.
+
+    :return:    Return ArgumentParser.
+    """
+
+    argparser = argparse.ArgumentParser(
+        description=__doc__,
```

Comment:
When printing the help in the command line, it will print the function docstring as a description. Not very helpful. What is the script doing?

Reply:
When doing python scripts -h ? 
The goal of this argument is to give to possibility to recompute data of all houses of yesterday

Reply:
Try running `python3 compute_power.py -h`. I don't believe the help will explain at all what the script does.